### PR TITLE
Create adding-trusted-committers.md

### PR DESCRIPTION
**Pull request by rrrutledge**
View [the document](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/rrrutledge-patch-2/docs/adding-trusted-committers.md).

### CODE REVIEW

File: docs/adding-trusted-committers.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+This document describes how to add a new Trusted Committer to the InnerSource Learning Path.
+
+1. Share your suggestion for new Trusted Commiter in the [:lock:learning-path-trusted-committers][learning-path-trusted-committers] private _Slack_ channel.
+2. If people seem to agree with your suggestion, then move forward.
+3. Create a pull request to the [Trusted Committers] section of the README with the new person.
+4. Invite the person into the [#learning-path-core] and [:lock:learning-path-trusted-committers][learning-path-trusted-committers] _Slack_ channels.
```
Comment (tsadler1988) [inline docs/adding-trusted-committers.md:6]: > #learning-path-core

Are we making use of this, or does the private TCs channel supersede it?

File: docs/adding-trusted-committers.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+This document describes how to add a new Trusted Committer to the InnerSource Learning Path.
+
+1. Share your suggestion for new Trusted Commiter in the [:lock:learning-path-trusted-committers][learning-path-trusted-committers] private _Slack_ channel.
+2. If people seem to agree with your suggestion, then move forward.
```
Comment (tsadler1988) [inline docs/adding-trusted-committers.md:4]: Could maybe explain this a bit more e.g.

```suggestion
2. If you have '[lazy consensus](https://community.apache.org/committers/lazyConsensus.html)' (e.g. 1 or more approval and 0 rejections within ~1 week of suggesting), then move forward.
```

File: docs/adding-trusted-committers.md (hunk 1 above)
Comment (rrrutledge) [reply docs/adding-trusted-committers.md:6]: I think we could delete the “-core” version.

Comment (tsadler1988) [reply docs/adding-trusted-committers.md:6]: > Does it really need the closed version?

This is potentially a bit sensitive. On the one hand, 'rejecting' a candidate TC, or having some other difficult discussion publicly, might not be good for psychological safety...but maybe having those discussions in the open will force people to be constructive with feedback e.g. "You're not quite ready for TC - here's what we'd like to see before granting you TC".

**Review by tsadler1988 — APPROVED** [review]

**Pull request merged**